compiler/luci/pass/src/UnrollUnidirectionalSequenceLSTMPass.cpp

```diff
@@ -162,6 +163,30 @@ luci::CircleTranspose *UnrollLSTM::first_transpose(luci::CircleNode *input)
   return transpose;
 }
 
+std::vector<luci::CircleUnpackOut *> UnrollLSTM::input_unpacks(luci::CircleNode *input)
```

Comment:
Parameter null checking seems needed. And so does below `unroll_lstm` method.

Reply:
I'll add null check in another PR.

Reply:
I think null check for input only in `unroll_lstm` will do.

Reply:
null check inside `input_unpacks` for safety seems nice.